docs/topics/gradle.md

```diff
@@ -225,55 +223,6 @@ Note that setting a toolchain via the `kotlin` extension will update the toolcha
 >
 {type="note"}
 
-To set any JDK (even local) for the specific task, use the Task DSL.
-
-### Setting JDK version with the Task DSL
```

Comment:
This section should be restored as this way still exists

Reply:
Can I throw this sentence in the `### Set custom JDK home` section in terms that we support Gradle 6.7.1+ now?
```
* For earlier Gradle versions without Java toolchains (up to 6.6) – with the [`UsesKotlinJavaToolchain` interface and the Task DSL](#setting-jdk-version-with-the-task-dsl).
```

And this 
`If you use a Gradle version earlier than 6.7, there is no [Java toolchains support](#gradle-java-toolchains-support).`
in the  `### Setting JDK version with the Task DSL` section.

Reply:
Yeah, we don't mention Gradle <6.7.1 anymore :smirk:

Reply:
Thanks, fixed!